stable/kommander/README.md

```diff
@@ -27,3 +27,17 @@ helm install --namespace "kommander" --name "kommander-kubeaddons" --set kommand
 ```bash
 helm install --namespace "kommander" --name "kommander-kubeaddons" --set kommander-cluster-lifecycle.certificates.issuer.selfSigned=true ./stable/kommander
 ```
+
+## Backporting / Patch Releases
```

Comment:
@joejulian @jr0d @jimmidysonI drafted up a readme with how to do a patch chart release -  as you guys have tackled chart patches in the past, would love your 👀 to make sure this process makes sense and is clear